### ISSUE TITLE
Lenovo Vantage / Hotkeys are uninstalled but still show active in LLT, and still seem active

### ISSUE THREAD

**Issue by brupec** [BartoszCichecki/LenovoLegionToolkit#1416]
### Rules

- [X] I made myself familiar with the README.
- [X] I checked closed issues for similar ones and did not find a similar problem mentioned before.
- [X] I understand that, if insufficient information was provided, my issue will be closed without an answer.

### Version

2.24.0

### OS

Windows 11 22H2 latest updates

### Device

Legion Slim 5 16IRH8

### BIOS version

M2CN25WW

### What's wrong?

Hi!

I tried to switch to full LLT, uninstalling lenovo legion toolkit and hotkeys.
I did first test with them switched off in LLT. Uninstalled them. Then when I unchecked in LLT I still have the warning that they run in background.
So I reinstalled them and uninstalled them again. Still the same problem.

I think it is true that they are still active, because I still have the lenovo hotkeys shortcut popups, and even if I disable them in LLT, the keyboard colors still show the one that were configured in vantage and not the one I configured in LLT.

I looked FAQ and readme and didn't find a clue on what can be wrong, or how to uninstall vantage and hotkeys completely.

Thanks in advance!

### How to reproduce the bug?

Install vantage and hotkeys.
Uninstall them
They show in LLT as active, keyboard RGB switch uses old vantage config, hotkeys shortcut shows lenovo style.
however if I disable hotkeys in LLT, it shows LLT style.
but if I disable vantage in LLT and reboot, it still shows vantage RGB

### What is the behavior that you expected?

Install vantage and hotkeys.
Uninstall them
They don't show in LLT as active and keyboardRGB / hotkeys uses LLT

### Logs

[log_2024_08_30_08_31_27.txt](https://github.com/user-attachments/files/16814128/log_2024_08_30_08_31_27.txt)

**Comment by brupec** [comment]
OK I got the idea to look at services and disable manually, that I should have had first.
I disabled ImControl and Hotkeys and now it's all working as expected.

So definately not LLT bug.

However, if you have an idea on how to properly uninstall those services, I'll take it :-)

Another question: should it be possible to have the tray icon of LLT change color depending if dGpu is active or not ? Would be nice to stay red for iGpu, and become green for dGpu, as LLT have the info. If you think it's good idea I can put a feature request.

Thanks for amazing tool btw
